DQMOffline/Trigger/python/HiggsMonitoring_Client_cff.py

```diff
@@ -1,4 +1,18 @@
 import FWCore.ParameterSet.Config as cms
 
-higgsClient = cms.Sequence(
+diphotonEfficiency = cms.EDAnalyzer("DQMGenericClient",
+    subDirs        = cms.untracked.vstring("HLT/Higgs/*"),
+    verbose        = cms.untracked.uint32(0), # Set to 2 for all messages
+    resolution     = cms.vstring(),
+    efficiency     = cms.vstring(
+        "eff_diphoton         'Photon turnON;             Photon pt [GeV]; efficiency'     photon_pt_numerator          photon_pt_denominator",
+        "eff_diphoton_variable 'Photon turnON;            Photon pt [GeV]; efficiency'     photon_pt_variable_numerator photon_pt_variable_denominator",
```

Comment:
where is the eff vs eta ?
I would suggest you to add the eff vs phi and eff vs eta-phi
it is important for the HEP 17